foo_uie_console/main.cpp

```diff
@@ -52,6 +52,9 @@ enum class EdgeStyle : int {
 cfg_int cfg_last_edge_style(GUID{0x05550547, 0xbf98, 0x088c, 0xbe, 0x0e, 0x24, 0x95, 0xe4, 0x9b, 0x88, 0xc7},
     static_cast<int>(EdgeStyle::None));
 
+cfg_bool cfg_last_append_newline(
+    {0x5db0b4d6, 0xf429, 0x4fc5, 0xb9, 0x1d, 0x29, 0x8e, 0xf3, 0x34, 0x75, 0x16}, static_cast<bool>(true));
```

Comment:
Doesn't look like the static cast is needed here:

```suggestion
    {0x5db0b4d6, 0xf429, 0x4fc5, 0xb9, 0x1d, 0x29, 0x8e, 0xf3, 0x34, 0x75, 0x16}, true);
```

(`EdgeStyle` is a scoped enum, so there's more casting involved for that...)

Reply:
Wow, I'm actually amazed to see this committed...